shared.csv

```diff
@@ -22,3 +22,5 @@ kustomize,Tools,Trial,TRUE,
 Kubernetes,Infrastructure Components,Trial,TRUE,
 Google Cloud Build,Services,Trial,TRUE,
 NextJS,Languages & Frameworks,Hold,TRUE,Benefits don't outweigh the effort for the time being
+JUnit 5,Languages & Frameworks,Trial,TRUE,
+GreenMail,Languages & Frameworks,Trial,TRUE,
```

Comment:
Wouldn't the `tools` category be a better fit for GreenMail?

Reply:
I thought so too but was unsure because that category only had CLI tools so far. But I'm glad you have the same feeling - I'll move it over.

Reply:
Thanks :)